core/packages/contracts/src/BeefyClient.sol

```diff
@@ -473,18 +491,45 @@ contract BeefyClient is Ownable {
         }
     }
 
+    // TODO: do we need to handle duplicate MMR_ROOT_ID entries in the payload?
+    // Unnecessary if we have a guarantee of unique payloadIDs per item.
+    // eg. if payload was a key-value mapping like a hash table.
```

Comment:
@vgeddes We're taking the first MMR root payload item for now. Guessing this is fine, as multiple MMR roots for the same (finalised) block number is invalid (and probably grounds for slashing).